crates/ra_analysis/src/job.rs

```diff
@@ -11,7 +11,7 @@ pub struct JobToken {
 }
 
 impl JobHandle {
-    pub fn new() -> (JobHandle, JobToken) {
+    pub fn new_pair() -> (JobHandle, JobToken) {
```

Comment:
Agree that `new` probably isn't the best name here, the whole `JobHandle` infra will be removed once we finish migration to salsa.